logger.js

```diff
@@ -54,17 +61,27 @@ function pinoLogger (opts, stream) {
       return
     }
 
-    log[level]({
-      res: this,
-      responseTime: responseTime
-    }, 'request completed')
+    var shouldLogSuccess = true
+    if (this._currentUrl && autoLoggingOptions && autoLoggingOptions.ignorePaths && autoLoggingOptions.ignorePaths.length) {
```

Comment:
There is no need to verify _all_ of these for every requests, as most should not change.

Reply:
fixed